core/core.go

```diff
@@ -70,15 +70,15 @@ func (h *Handler) info(ctx context.Context) (map[string]interface{}, error) {
 
 func (h *Handler) leaderInfo(ctx context.Context) (map[string]interface{}, error) {
 	var (
-		generatorHeight  *uint64
-		generatorFetched *time.Time
+		generatorHeight  uint64
```

Comment:
That field would appear as 0 under the same circumstances as it would show up as null before this patch.

Based on the comments below, it happens when we're having trouble connecting to the generator. Looks like this patch should also update the last sentence of the comment on line 91:

```
// fetchTime might be the zero time if we're having trouble connecting
// to the remote generator. Only set the height & time if we have it.
// The dashboard will handle nulls correctly.
```